cves/CVE-2014-0473.yml

```diff
@@ -306,7 +293,14 @@ mistakes:
     Use those questions to inspire your answer. Don't feel obligated to answer
     every one. Write a thoughtful entry here that those ing the software
     engineering industry would find interesting.
-  answer:
+  answer: |
+    Coding mistakes and design mistakes were made attributing to this vulnerabilty.
+    If the developers designed the caching framework to handle the anonymous users
+    properly, the coding mistake wouldn't have been made. Therefore it is a combination
```

Comment:
I still think this is a design flaw and not a coding mistake, feel free to disagree.